docroot/themes/custom/mass_theme/mass_theme.theme

```diff
@@ -4377,6 +4377,59 @@ function mass_theme_preprocess_field__person_org_role(&$variables) {
   $variables['person_org_roles'] = $person_roles;
 }
 
+/**
+ * Builds alert blocks.
+ */
+function mass_theme_build_alert_blocks_sooner() {
+  // Retrieve active theme.
+  $theme = \Drupal::theme()->getActiveTheme()->getName();
+
+  // Retrieve theme regions.
+  $available_regions = system_region_list($theme, 'REGIONS_ALL');
+
+  // For each region.
+  foreach ($available_regions as $key => $region) {
+
+    // Load region blocks.
+    $blocks = \Drupal::entityTypeManager()
+      ->getStorage('block')
+      ->loadByProperties(['theme' => $theme, 'region' => $key]);
+
+    /** @var \Drupal\block\Entity\Block $block */
+    foreach ($blocks as $key => $block) {
+      if (!$block->access('view')) {
+        continue;
+      }
+      if ($block->getPlugin()->getBaseId() != 'mass_alerts_block') {
+        continue;
+      }
+      $block->getPlugin()->build();
+    }
+  }
+}
+
+/**
+ * Data to fetch the alerts as soon as possible.
+ *
+ * @see docroot/themes/custom/mass_theme/templates/layout/html.html.twig
+ * @see Drupal\mass_alerts\Plugin\Block:build()
+ */
+function mass_theme_fetch_alerts_sooner(&$variables) {
```

Comment:
Lets rename this. This gets $variables, it doesnt do the fetch.

Reply:
Sure! Will do.